packages/jest-circus/src/utils.js

```diff
@@ -222,14 +225,16 @@ export const getTestDuration = (test: TestEntry): ?number => {
 export const makeRunResult = (
   describeBlock: DescribeBlock,
   unhandledErrors: Array<Error>,
+  config: ProjectConfig,
 ): RunResult => {
   return {
-    testResults: makeTestResults(describeBlock),
+    testResults: makeTestResults(describeBlock, config),
     unhandledErrors: unhandledErrors.map(_formatError),
   };
 };
 
-const makeTestResults = (describeBlock: DescribeBlock): TestResults => {
+const makeTestResults = (describeBlock: DescribeBlock, config): TestResults => {
+  const stackUtils = new StackUtils();
```

Comment:
You can instantiate this at the top of the file. It will always be instantiated anyways